internal/component/discovery/process/args.go

```diff
@@ -29,6 +30,7 @@ var DefaultConfig = Arguments{
 		Exe:         true,
 		Commandline: true,
 		ContainerID: true,
+		CgroupPath:  true,
```

Comment:
I would argue that we should disable it by default. As without further config it is not a useful metadata.

When you really need it then you can also set this to `true`, when adding your `disocvery.relabel`.

Wdyt?

Reply:
Fair enough. Makes sense!